iocage_cli/start.py

```diff
@@ -21,34 +22,160 @@
 # STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
 # IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-"""start module for the cli."""
+"""Start jails from the CLI.."""
```

Comment:
what's with the dot-dot `..`?

Reply:
That's a typo! (MyPy should complain about it though)